sdks/python/apache_beam/utils/annotations_test.py

```diff
@@ -32,9 +32,12 @@ def test_deprecated_with_since_current_message(self):
       @deprecated(since='v.1', current='multiply', extra_message='Do this')
       def fnc_test_deprecated_with_since_current_message():
         return 'lol'
+      # In python2, the test framework by default adds an ignore filtering rule.
```

Comment:
Thanks. Turns out DeprecationWarnings are ignored by default, not just on Python 2, see: https://docs.python.org/3/library/warnings.html. So I suggest the following wording (let's add a clarifying comment for all test methods that need this).
 
```
def test_deprecated_with_since_current_message(self):
  with warnings.catch_warnings(record=True) as w:
    # Deprecation warnings are ignored by default, turn them on to enable testing.
    warnings.simplefilter("once", DeprecationWarning)
```
Adding this to each individual test is the right thing, since we add a filter override inside `catch_warnings` context manager, so it will not affect other tests in the suite.

Reply:
Acked.  Thanks.

Though this PR is now in conflict with another non-trivial merge.  Does not seem trivial to get all resolved. 

Please let me know if there is any way better than copy over the master version and apply all my changes again.